Assets/00_Content/Scripts/Interactables/Table.cs

```diff
@@ -48,5 +61,23 @@ public class Table : Interactable {
 
 			return closest != null;
 		}
+
+		public void Damage(float damage) {
+			if (IsDestroyed) return;
+
+			health = Mathf.Max(0, health - damage);
+			if (IsDestroyed) {
+				GetComponentInChildren<MeshRenderer>().enabled = false;
+				if (Tavern.Instance != null)
+					Tavern.Instance.TakesDamage(1);
+			}
+		}
+
+		public void Repair() {
+			if (RoundController.Instance != null)
+				health = RoundController.Instance.CurrentDifficulty.tableHealth;
+
```

Comment:
So, considering you send to function Tavern.Instance.TakesDamage() on line 72 when/if the table is destroyed, should you call the function Tavern.Instance.Repair() here as well?